content/en/tutorials/ocf-conformance-testing.md

```diff
@@ -174,3 +178,461 @@ The steps needed to compile and run `cloud_tests` device with plgd hub #bundle's
     Input the option number to issue the selected command to the binary.
 
 ## Validating C2C Origin Cloud Conformance
+
+The Origin Cloud API is implemented by the `cloud2cloud-connector` service, which is part of the [plgd hub #bundle](https://plgd.dev/quickstart/deploy-plgd-hub/#plgd-bundle). The public API can be examined at <https://github.com/plgd-dev/hub/blob/main/cloud2cloud-connector/swagger.yaml>.
+Support of the API in the plgd hub #bundle GUI is planned, but not yet implemented. So for now you need an application or a utility that allows you to send HTTP requests with custom headers. In this guide I will use the [Postman](https://www.postman.com/) application.
+
+Additional, several of the Origin Cloud test cases require different [setup](https://plgd.dev/configuration/cloud2cloud-connector) of the `cloud2cloud-connector` and also different values in the PICS configuration. Only with the correct configuration of each, the test cases will succeed.
+
+### Executing Origin Cloud API commands using Postman
+
+After starting the bundle as described in [Run plgd hub #bundle in Ubuntu machine](#run-plgd-hub-bundle-in-ubuntu-machine) the `cloud2cloud-connector` service is available at the address ```https://{FQDN}/c2c/connector```.
+
+#### Prepare environment
+
+In the sidebar select **Environment**, open the **Globals** pane and setup three variables there:
```

Comment:
typo - pane

Reply:
What word are you thinking of? Panel? 

Pane is a word (https://www.merriam-webster.com/dictionary/pane) though, I've used it as synonym for section.

Reply:
LOL, okay :) My fail.